vmdb/app/views/miq_request/_prov_vc_grid.html.haml

```diff
@@ -15,11 +15,6 @@
               %img{:src => "/imgas/icons/16/sort_#{@edit[:vc_sortdir] == 'ASC' ? 'up' : 'down'}"}
     %tbody
       - cls = @edit[:new][:sysprep_custom_spec] && @edit[:new][:sysprep_custom_spec][0].nil? ? "row3" : cycle('row0', 'row1')
-      %tr{:class => cls, :onclick => "miqAjax('/miq_request/prov_field_changed/?#{field_id}=__VC__NONE__&id=#{id}');", :nowrap => 1}
-        %td{:onmouseover => "ChangeColor(this, true);", :onmouseout => "ChangeColor(this, false);"}
-          = "<#{_('None')}>"
-        %td
-        %td>
```

Comment:
This change removes `None` from the list unconditionally.  The thought was to remove `None` from the list if the field is marked as required.